eva-accession-ws/src/main/java/uk/ac/ebi/eva/accession/ws/rest/VariantAccessioningRestController.java

```diff
@@ -17,19 +17,30 @@
  */
 package uk.ac.ebi.eva.accession.ws.rest;
 
+import io.swagger.annotations.ApiOperation;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
-import uk.ac.ebi.eva.accession.core.SubmittedVariantAccessioningService;
-import uk.ac.ebi.eva.accession.core.ISubmittedVariant;
 import uk.ac.ebi.ampt2d.commons.accession.rest.BasicRestController;
 
+import java.util.List;
+import java.util.Map;
+
 @RestController
-@RequestMapping(value = "/v1/variant")
-public class VariantAccessioningRestController extends BasicRestController<ISubmittedVariant, SubmittedVariantDTO, Long> {
+@RequestMapping(value = "/v1/submitted-variants")
+public class VariantAccessioningRestController {
+
+    private final BasicRestController basicRestController;
```

Comment:
have you checked that this actually hides the post endpoint? I would expect so, but just in case